TESTS/network/wifi/wifi_connect_nocredentials.cpp

```diff
@@ -28,6 +28,5 @@ void wifi_connect_nocredentials(void)
     WiFiInterface *wifi = get_interface();
     nsapi_error_t error;
     error = wifi->connect();
-    wifi->disconnect();
-    TEST_ASSERT(error == NSAPI_ERROR_PARAMETER);
+    TEST_ASSERT(error == NSAPI_ERROR_NO_SSID);
```

Comment:
Should we also accept the `NSAPI_ERROR_PARAMETER`

I'm not sure that all drivers currently return this NO_SSID.

Reply:
I'll add it.